src/test/java/edu/cornell/kfs/sys/service/impl/AwsSecretServiceImplTest.java

```diff
@@ -0,0 +1,107 @@
+package edu.cornell.kfs.sys.service.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.text.ParseException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import edu.cornell.kfs.sys.service.impl.fixture.AwsSecretePojoBasic;
+
+class AwsSecretServiceImplTest {
```

Comment:
This test appears to be dependent on an unmocked AWS Secret service being available. Should this class have a suffix such as "IntegTest" to indicate that it's actually more of an integration test?

Reply:
Ok, I opted for IntegrationTest